src/harness/fourslash.ts

```diff
@@ -3720,6 +3740,10 @@ namespace FourSlashInterface {
             this.state.verifySymbolAtLocation(startRange, declarationRanges);
         }
 
+        public typeOfSymbolAtLocation(range: FourSlash.Range, symbol: ts.Symbol, expected: string) {

```

Comment:
Well, quick info gets us the type of the symbol under the cursor -- this lets us call `getTypeOfSymbolAtLocation` on *any* symbol, at the current location.